Update/omot_011.txt

```diff
@@ -58,7 +58,7 @@ void main()
 			NULL, " I'm sure Mion will be able to take care of it.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100968", 256, TRUE);
 	OutputLine(NULL, "一人だとろくな考えにならないけど、レナも魅音だったら、色々と話しやすいだろうしさ」",
-			NULL, " It would be tough to handle on her own, but Mion's there with her so they'll be able to talk things through.\" ", Line_Normal);
+			NULL, "Her isolation would make her opinion so bad, but Mion's there with her so they'll be able to talk things through.\" ", Line_Normal);
```

Comment:
```suggestion
			NULL, " It would be tough to handle on her own, but Mion's there with her so they'll be able to talk things through.\" ", Line_Normal);
```